interface/ceed-basis.c

```diff
@@ -211,7 +217,7 @@ int CeedBasisCreateH1(Ceed ceed, CeedElemTopology topo, CeedInt ncomp,
   ierr = CeedMalloc(dim*Q*P,&(*basis)->grad1d); CeedChk(ierr);
   memcpy((*basis)->interp1d, interp, Q*P*sizeof(interp[0]));
   memcpy((*basis)->grad1d, grad, dim*Q*P*sizeof(grad[0]));
-  ierr = ceed->BasisCreateH1(ceed, topo, dim, P, Q, interp, grad, qref,
+  ierr = ceed->BasisCreateH1(topo, dim, P, Q, interp, grad, qref,
```

Comment:
Is it interesting to pass all these arguments along when they have already been set on the `CeedBasis` above? Seems like maybe we should only do that for args that only the backend can handle (e.g., with `CeedMemType`).

Reply:
Probably not. I left it to make the first commit simpler but I can trim other arguments too. Eventually I'd like the backends to not reach directly into the private struct to get that data, but that's not related to this PR

Reply:
Maybe in that 'to come soonish' PR I make a round of getters for those values and convert the backends over to using them while also only passing arguments that are needed. Those two make sense together for me.